tmt/utils/__init__.py

```diff
@@ -1166,6 +1175,8 @@ def run(
             interaction with user.
         :param timeout: if set, command would be interrupted, if still running,
             after this many seconds.
+        :param on_process_end: if set, this callable would be called after the
+            command process finishes.
```

Comment:
Nitpick: Should probably go under the `on_process_start` description below.

Reply:
So, I guess, what's remaining is just this nitpick and the release note. @happz, or do you still plan to extend the test coverage and docs as the checklist suggests?

Reply:
Both addressed in 72ab223b.

I do not plan to add any more touches for now, there is still work left, and I think adding a paragraph or two describing what tmt does when interrupted would be better when the task is complete.